src/main.js

```diff
@@ -1,159 +1,164 @@
-        // add class, id, index
-        var self = $(item),
-            id = item.id ? '#' + item.id : '',
-            // gets all the classes for an item, and chains them together
-            clss = item.classList.length ? item.classList.toString().split(' ').map(function (c) {
-                return '.' + c;
-            }).join('') : '',
-            name = item.nodeName.toLowerCase(),
-            index = self.siblings(name).length ? ':nth-child(' + (self.index() + 1) + ')' : '';
-        // Check if the name is html or body, which are returned immediately
-        if (name === 'html' || name === 'body') {
-            return name;
-        }
-        // Other elements are returned with their index, id, and classes
-        return name + index + id + clss;
-    // Shows parent-child relationship
-    }).join('>');
-    return quickCss;
-};
-
-$(document).on('mouseenter', '*', function(e) {
-  // $(this).stop().animate({'border': '3px solid yellow'});
-  $(this).addClass('liveAPI-border-yellow');
-  e.stopImmediatePropagation();
-})
-
-$(document).on('mouseleave', '*', function(e) {
-  $(this).removeClass('liveAPI-border-yellow');
-  // $(this).stop().animate('border', 'none');
-  e.stopImmediatePropagation();
-})
-
-// click on part of the page to see the CSS selector
-$(document).on('click','*', function(){
-    console.log('DOM Path', $(this).fullSelector() );
-    return false;
-});
-
-/* Avoid selection of parent elements
-    - select div elements that do not have child divs
-    - extract a, li elements
-
-    Select all elements w/ the same path
-      - upon x, remove extra elements selected
-*/
-
-// Highlighting functionality for a selected HTML element
-// Create a div "overlay" that contains original text and exit button
-$(document).on('click', '*', function(e) {
-  if ($(e.target).is('div')) {
-    // console.log($(this).text());
-  }
-})
-
-$(document).on('click', '.liveAPI-highlight', function(e) {
-  e.stopImmediatePropagation();
-});
-
-$(document).on('click', '.liveAPI-highlight-wrapper', function(e) {
-  e.stopImmediatePropagation();
-});
-
-$(document).on('click', '.liveAPI-highlight-button', function(e) {
-  $(this).parent().remove();
-  // prevents other listeners of the same event from being called
-  e.stopImmediatePropagation();
-})
-
-$(document).on('click', '*', function() {
-  let children = $(this).children().map((i, ele) => {
-    return ele.nodeName.toLowerCase();
-  }).get();
-  // Check if this is a parent div, with child div elements
-  console.log('html element', $(this));
-  if ($(this)[0].nodeName.toLowerCase() === 'div' && children.includes('div')) return false;
-  let styles = $(this).css([
-    "width", "height", "font-size", "font-weight", "font-family", "font-variant", "font-stretch", "line-height", "text-transform", "text-align", "padding-top", "padding-bottom", "padding-left", "padding-right", "letter-spacing"]
-  );
+//   // Logic for getting the DOM Path for a selected HTML Element
```

Comment:
Don't need to leave all of this commented code in here. We can remove it next PR.